test/bot.test.js

```diff
@@ -282,6 +282,13 @@ describe('Bot', function () {
     this.bot.parseText('<!somecommand> <!somecommand|readable>')
       .should.equal('<somecommand> <readable>');
   });
+  it('should handle entity-encoded messages from slack', function () {
```

Comment:
woop, this should have a newline before it :D 
would you mind adding an explicit assertion that makes sure [this](https://github.com/ekmartin/slack-irc/issues/160#issuecomment-249626924) doesn't happen anymore as well?